sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -845,6 +846,7 @@ private[hive] class HiveClientImpl(
     table.comment.foreach { c => hiveTable.setProperty("comment", c) }
     table.viewOriginalText.foreach { t => hiveTable.setViewOriginalText(t) }
     table.viewText.foreach { t => hiveTable.setViewExpandedText(t) }
+    table.viewDefaultDatabase.foreach {t => hiveTable.setProperty("viewDefaultDatabase", t)}
```

Comment:
First, we should define `viewDefaultDatabase ` in the object `HiveExternalCatalog`. See the [examples](https://github.com/apache/spark/blob/master/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala#L1059-L1083).

Second, we do not convert the table properties here. Normally, all the table properties should be converted in [`restoreTableMetadata`](https://github.com/apache/spark/blob/master/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala#L615).